docs/db/db-relations.md

```diff
@@ -0,0 +1,191 @@
+
+1. One end of the line, refers to the **max** number of times that an instance of one entity can be associated with instances in the related entity. It can be **one**, **many** or **none**
+
+2. The second, describes the **min** number of times one instance can be related to others.
+It can be **zero** or **mandatory**
+
+The combination of those two indicators create relations, you can get more information [here](https://community.mis.temple.edu/mis3506digitaldesignfall2018/files/2018/10/Adam-Alalouf_Cardinality.pdf)
+
+![alt text](../img/db/db-relations.jpg "hierarchical roles")
+
+## Relations
+
+### Events
+
+Events will be the accumulation of the following tables :
+
+- `event`
+- `registration`
+- `registration_user`
+- `event_user`
+
+This part of the database takes care of schools activities with time.
+
+The `event` table contains the following relations:
+
+- `parentId` that is associated to himself. This association can be better explained if we take a look at the `path` column. An event can be based on a parent event (ex: the event `/madere/div-01/piscine-rust` is associated to this parent event `/madere/div-01`).
+
+- `registrationId` that is associated to the `registration` table. An event can have multiple registrations, but a registration can only be associated to one event, this being the parent event of the registration table.
+
+- `objectId` that is associated to the `object` table, this column will identify which object is associated to which event. An object can have multiple events but must have at least one.
+
+---
+
+The `event_user` contains the associations between events and users. This will contain information on which users are registered to which event. An event can have multiple users and a user can have multiple events, this way being a many to many relationship.
+
+---
+
+The `registration` table contains all objects/activities that need registration. Having the following relations:
+
+- `parentId` that is associated to himself. As said above (in the events), it has the same use.
+- `objectId` that is associated to the object table, this is the object reference to the registrations.
+
+---
+
+The `registration_user` table is a relationship many to many between `registration` and `user`. This table contains all users that are registered to an event.
+
+- `registrationId` a registration can have multiple users.
+- `userId` a user can have multiple registrations.
+
+---
+
+### Objects
+
+Object will be the accumulation of the following tables :
+
+- `object`
+- `object_child`
+- `object_status`
+- `object_type`
+
+This part of the database defines the structure of the content.
+
+The `object` table contains the following relations:
+
+- `referenceId` that is associated to himself, an object can have none or many references. This attribute allows the creation of duplicates and allows each campus to create their own objects.
```

Comment:
an object can't have many references, it only has one. But it can be the reference of multiple objects :)
in here I think this would be clearer: "This attribute allows the duplication of reference objects (given by 01-edu), that are used to set up campuses"

Reply:
nice thank you :)